src/components/ServerList.tsx

```
@@ -126,1062 +109,896 @@ const themeLightWarm = themeQuartz.withPart(colorSchemeLightWarm);
+                tagChangerValueCallback: (tag: any) => {
+                  return nameFilters["t-" + tag.docsName];
+                },
+                categoryChangerValueCallback: (categorie: any) => {
+                  return nameFilters["c-" + categorie.name];
+                },
+                categoryChangerCallback: (categorie: any) => async (b) => {
+                  var filt = nameFilters;
+                  filt["c-" + categorie.name] = b;
+                  setNameFilters(filt);
+                  if (b) {
+                    var filt2 = filters;
```

Biome (1.9.4)

[error] This var should be declared at the root of the enclosing function.

The var is accessible in the whole body of the enclosing function.
To avoid confusion, it should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)